test/test_tweets_dump.py

```diff
@@ -0,0 +1,73 @@
+# -*- coding: utf-8 -*-
+from __future__ import absolute_import
+
+import os
+import sys
+import unittest
+sys.path.insert(0, os.path.abspath(
+    os.path.join(os.path.dirname(__file__), '..')))
+from resources.tweets_dump import TweetsDump
+
+'''
+    @license
+
+    Copyright(c) 2018, GarridoLabs and the project's contributors.
+
+    This source code is licensed under the Apache License, Version 2.0 found in
+    the LICENSE.txt file in the root directory of this source tree.
+'''
```

Comment:
The license header should be placed as high as possible. Ideally, after the `# -*- coding: utf-8 -*-` statement.